lib/matplotlib/axes/_base.py

```diff
@@ -3356,6 +3350,7 @@ def get_ylim(self):
         """
         return tuple(self.viewLim.intervaly)
 
+    @munits._accepts_units(convert_y=['bottom', 'top'])
```

Comment:
I just wanted to note that this fixes a bug for us where we ran into problems setting limits using units. (Note the lack of a call to `_process_unit_info()` in the original `set_ylim()` code.)